library/tasks/StarAlignBamSingleEnd.wdl

```diff
@@ -8,6 +8,7 @@ task StarAlignBamSingleEnd {
   Int input_size = ceil(size(bam_input, "G"))
   Int reference_size = ceil(size(tar_star_reference, "G"))
   Int estimated_disk_required =  ceil(input_size * 2.2 + reference_size * 2)
+  Int estimated_memory_required = reference_size + 6
```

Comment:
Which is larger? 30 GB or `reference_size + 6` GB?

Reply:
Depends on reference_size. :)